units/Unit1.txt

```diff
@@ -0,0 +1,3 @@
+Kakarott
+4455
+113
```

Comment:
Ezeket a fajlokat azonnal torolni.

Reply:
A json fajlok nem disznek vannak ott.

Reply:
@hegyhati

TL;DR: a parser script régóta kész és működik
______________________________________
A legtöbb kódrészlet kommentre válasz:

1. A "kódszemét" részeket eredményező és a Unit .txt-ket hozzáadó commit nevében is jeleztük, hogy a pythonban írt parser bypass-elése miatt van. Ezt azért csináltuk, hogy az egész program a workflowban is lefusson. A parser a GitHub Actions-el ellentétben más Linuxos felületen tökéletesen futott !!már a határidő előtt is!!. Csak itt nem, ezért is kértünk segítséget. A workflow setupolása is tökéletesen működött, azt is kipróbáltuk külön. (Persze ha az egészet megvalósíthatnánk, akkor a workflow .yml-ét is javítanánk.)

2. A többi megjegyzés már az előző körben elfogadott kódrészletekre vonatkoznak, amiket persze azonnal tudnánk javítani.
______________________________________
Ma kaptuk a segítséget tartalmazó e-mailt, és abban nem volt szó lezárásról, sőt, mintha a folytatásra bíztatna. Kicsivel rá (a kettő között nem volt alkalmunk a javításra) került lezárásra a repó fő okként a parser hiányát megadva, pedig az létezett. Ezek figyelembe vételével kérném, hogy ahogyan a délutáni emailben is állt, legyen lehetőségünk a harmadik feladat kiírásáig javítani véglegesre hozni a második feladat branchét.